pkg/ebpf/events_processor.go

```diff
@@ -440,5 +453,23 @@ func (t *Tracee) processTriggeredEvent(event *trace.Event) error {
 	// will be moved back as such we apply the value internally and not
 	// through a referene switch
 	(*event) = withInvokingContext
+	if event.EventID == int(events.PrintMemDump) {
```

Comment:
suggestion: You should do this in a separate processor function. You can now register multiple processors for one event :)